vnext/Desktop/DevSupportManager.cpp

```diff
@@ -55,60 +55,50 @@ string DevSupportManager::GetJavaScriptFromServer(
     const string &platform) {
   auto bundleUrl =
       DevServerHelper::get_BundleUrl(debugHost, jsBundleName, platform /*platform*/, "true" /*dev*/, "false" /*hot*/);
-
-  Url url(bundleUrl);
-  auto const resolveResult = m_resolver.resolve(url.host, url.port);
-  tcp::socket socket{m_context};
-  boost::system::error_code ec;
-  connect(socket, resolveResult, ec);
-  if (ec) {
+  Uri uri(Common::Unicode::Utf8ToUtf16(bundleUrl));
+  Filters::HttpBaseProtocolFilter filter;
+  filter.CacheControl().ReadBehavior(Filters::HttpCacheReadBehavior::NoCache);
+  HttpClient client(filter);
+  client.DefaultRequestHeaders().Host(HostName(uri.Host().c_str()));
+
+  HttpRequestMessage request(HttpMethod::Get(), uri);
+  HttpResponseMessage response = client.SendRequestAsync(request).get();
+
+  IBuffer buffer0 = response.Content().ReadAsBufferAsync().get();
```

Comment:
>buffer0 [](start = 10, length = 7)

(nit) why buffer0? are we expecting to have more buffers?

Reply:
No. Previous boost-based overlapping variable names. They have been removed. Fixing...